CONTRIBUTING.md

```diff
@@ -301,6 +301,17 @@ To manually create a backport pull request, follow [the standard pull request pr
 - When the backport pull request is merged and you have the necessary privileges you can also replace the label `9.needs: port to stable` with `8.has: port to stable` on the original pull request.
   This way maintainers can keep track of missing backports easier.
 
+## How to request review of pull requests
+[pr-review-request]: #how-to-request-review-of-pull-requests
+
+Due to the large number of pull requests that need to be handled, some of them may be overlooked by the reviewer, may not receive feedback after completing the requested changes, or may be ignored by committers and remain unmerged for a long time. You can remedy this situation but notifying reviewers and committers through the following channels:
+
+- [NixOS Discourse: PRs ready for review](https://discourse.nixos.org/t/prs-ready-for-review/3032) for pull requests not yet receiving any reviews, already approved or made requested changes.
+
+- [Matrix Room: Nixpkgs Review Requests](https://matrix.to/#/#review-requests:nixos.org) for pull requests that have been waiting for more than a week in the above situations.
```

Comment:
Additional suggestion: Forbid Cc teams for requesting reviews as found in https://github.com/NixOS/nixpkgs/pull/342437

This creates unnecessary overloads on most participants and GitHub servers.

Reply:
I believe requesting reviews from GitHub teams are occasionally useful, like when the systemd or security teams get requested for their collective acknowledgement.